app/i18n/locales/en-US.json

```diff
@@ -46,7 +46,7 @@
   "daedalusTransfer.instructions.attention.masterKey.button.label": "Daedalus Master Key",
   "daedalusTransfer.instructions.attention.paper.button.label": "Daedalus Paper Wallet",
   "daedalusTransfer.instructions.attention.text": "Check the balance after the <strong>first snapshot</strong> (dry run) for a <strong>Daedalus Wallet</strong>. <br /> This first Testnet only has the purpose to help users assure that their balance got copied correctly.<br /> As a result, you will be able to see what was the balance that got copied from Mainnet to the Testnet. This balance should match your holdings on November 12th.",
-  "daedalusTransfer.instructions.attention.title": "Transfer all funds from",
```

Comment:
You need to change this string for all other languages also by setting them to the English version